clients/java/src/main/java/io/camunda/zeebe/client/impl/worker/JobWorkerImpl.java

```diff
@@ -219,13 +223,24 @@ private void backoff(final JobPoller jobPoller, final Throwable error) {
   }
 
   private void handleJob(final ActivatedJob job) {
+    metrics.jobActivated(1);
```

Comment:
:thought_balloon: There's some clear duplication with the handlers here, primarily the choice of the done callback. I wanted to keep the changes as simple as possible for now, but if you'd like we can tackle it here immediately.